Pipfile

```diff
@@ -4,33 +4,33 @@ verify_ssl = true
 name = "pypi"
```

Comment:
I guess we already had a pipenv/pipfile, but I don't think we typically used it 🤷 

This updates the dependencies to the current versions and adds some scripts for testing